scripts/populate_ingest/services/ingest.py

```diff
@@ -71,3 +71,9 @@ def project_exists(self, uuid: str) -> bool:
         results = response.json()
         count = results.get('page', {}).get('totalElements', 0)
         return count > 0
+
+    def get_all_projects(self):
```

Comment:
There's a method call `get_all` from ingest_api.py . Do note that it returns an iterator and not a list.

Reply:
would you recommend using that and removing `get_all_projects`?

Reply:
yeah, i would prefer using `get_all`.

Reply:
+1, disregard my comments on that. forgot about that!

Reply:
ah yours has the same implementation as get_all i think :)